bika/lims/browser/bika_listing.py

```diff
@@ -594,7 +594,7 @@ def _process_request(self):
                     continue
                 ##logger.info("Or: %s=%s"%(index, value))
                 if idx.meta_type in('ZCTextIndex', 'FieldIndex'):
-                    self.Or.append(MatchRegexp(index, value))
+                    self.Or.append(MatchRegexp(index, '%s*' % value))
```

Comment:
Thanks for reviewing this Jean. MatchRegexp seems to handle * correctly and when it makes no difference to the result if I use MatchGlob. BTW, this works well for text but not so well for numbers. This client has strings like "1,1,1-Methyl Hydride" but advanced query is not a friend of numbers. Any ideas?